index.html

```diff
@@ -5500,6 +5500,44 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*CZ's card END ________  -->
 
+                <!-- ============================================================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+        <!-- make sure there is one line of space above and below your card -->
+
+        <!-- ________ *mirfanrafif card START ________  -->
```

Comment:
Looks like you have added your card at the end of file, you have to add your card just below the template as per instructions give in step 5, you can check it [here](https://github.com/Syknapse/Contribute-To-This-Project/).
Once the changes are done, I'll review the PR and merge it.